charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-node.yaml

```diff
@@ -1,3 +1,31 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-azuredisk-node-config
```

Comment:
We should use the name `{{ .Values.linux.dsName }}-config` to enable side-by-side installs in the future.